checker/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="generator" content="HTML Tidy for HTML5 for Apple macOS version 5.7.16">
```

Comment:
nit: please remove this

Reply:
Seems a bit rude to the authors of htmltidy to not give them credit...but done.